bugToJira.py

```diff
@@ -0,0 +1,258 @@
+
+# We will store a dict with bugzilla {bug id: jira issue} to, then, add
+# android versions, comments and attachments
+bug_id_jira_issue_dict = {}
+
+# Create JIRA issues from bugs (without comments, attachments and Android versions)
+print("Creating JIRA issue for each bug...\n")
+for row_bug in rows_bugs:
+    print("Loading bug {0} : {1}".format(row_bug[0], row_bug[4]))
+    issue_summary = '[BZ' + str(row_bug[0]) + '] ' + row_bug[4]
+    issue_description = '[CREATED]: ' + str(row_bug[3]) + '\n' + \
+                        '[Operating system]: ' + row_bug[5] + '\n' + \
+                        '[Product]: ' + row_bug[7] + '\n' + \
+                        '[Platform]: ' + row_bug[8] + '\n' + \
+                        '[Version]: ' + row_bug[9] + '\n' + \
+                        '[Android Version]: ' + ANDROID_VERSION_KEY + '\n' + \
+                        '[Component ID]: ' + row_bug[10] + '\n' + \
+                        '[Affected users]: ' + row_bug[12] + '\n' + \
+                        '[Zendesk ticket]: https://' + zendesk_instance + \
+                            '.zendesk.com/agent/tickets/' + row_bug[13] + '\n' + \
+                        '[Description]:\n' + DESCRIPTION_KEY
+    issue_priority = row_bug[6].replace("Normal", "Medium").replace("---", "Medium")
+    issue_dict = {'project':jira_project,
+                  'summary': issue_summary,
+                  'description': issue_description,
+                  'priority': {'name': issue_priority},
+                  "labels": [JIRA_LABEL]}
+    issue_type = 'Story'
+    if row_bug[11] in ['Ticket', 'Problem']:
+        issue_type = 'Bug'
+        issue_severity = row_bug[1].replace("---", "normal").replace("-", " ")
+        issue_status = row_bug[2].replace("---", "TO_CHECK") \
+                                 .replace("-", " ") \
+                                 .replace("_", " ") \
+                                 .capitalize()
+        issue_dict.update({'customfield_10600': {'value': issue_severity},
+                           'customfield_10601': {'value': issue_status}})
+    issue_dict.update({'issuetype': {'name': issue_type}})
+    new_issue = jira.create_issue(fields=issue_dict)
+    bug_id_jira_issue_dict.update({row_bug[0]: new_issue})
+    print("Created JIRA issue https://" + jira_instance + \
+          ".atlassian.net/browse/" + new_issue.key + "\n")
+
+# Add Android versions for each issues
+print("Adding Android versions to JIRA issues...")
+# Create a dict with bug_id | android versions
+bug_id_android_versions_dict = {}
+# Loop on android versions table to fill the dict
+for row_android_version in rows_android_version:
+    value = bug_id_android_versions_dict.get(row_android_version[0])
+    if value == None:
+        bug_id_android_versions_dict.update({row_android_version[0]: row_android_version[1]})
+    else:
+        bug_id_android_versions_dict.update({row_android_version[0]: value + ", " + row_android_version[1]})
```

Comment:
The if could be simplified as:

    if value is None:
        bug_id_android_versions_dict[row_android_version[0]] = value
    else:
        bug_id_android_versions_dict[row_android_version[0]] = value + ", " + row_android_version[1]

Or did I miss something?